pkg/conda/meta/testdata/README.md

```diff
@@ -0,0 +1,30 @@
+To recreate test files, run:
+```bash
+conda create --yes -n test-dep-parser python=3.8.8
+cp ~/miniconda3/envs/test-dep-parser/conda-meta/*
```

Comment:
When i tested this - i got different filepath:
```
(base) root@461420cfb7e0:~# find / -name "test-dep-parser"
/opt/conda/envs/test-dep-parser
```

can you check this?

Reply:
`~/miniconda3` is where a generic installation of conda stores environments while `/opt/conda` is where the official conda container stores them. Since test files are now generated from the container image I updated the paths in the doc to `/opt/conda`.